src/fides/api/api/v1/endpoints/system.py

```diff
@@ -396,16 +400,24 @@ async def ls(  # pylint: disable=invalid-name
     Otherwise all Systems will be returned (this may be a slow operation if there are many systems,
     so using the pagination parameters is recommended).
     """
-
-    query = select(System)
+    if not (
+        size
+        or page
+        or search
+        or data_uses
+        or data_categories
+        or data_subjects
+        or dnd_relevant
+        or show_hidden
+    ):
+        return await list_resource(System, db)
```

Comment:
i'd leave a comment here to prevent this type of back and forth in the future: 
```suggestion
        # if no advanced parameters are passed, we return a very basic list of all System resources
        # to maintain backward compatibility of the original API, which backs some important client usages, e.g. the fides CLI
        
        return await list_resource(System, db)
```